w1/zadania/swagrzyk/zadanie1.java

```diff
@@ -0,0 +1,29 @@
+public class zadanie1 {
+
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+        int c = 3;
+        int d = 4;
+        int ab = a + b;
+        int cd = c - d;
+        double e = 1.1;
+        double f = 2.2;
+        double g = 3.3;
+        double h = 4.4;
+        double ef = e * f;
+        double gh = g / h;
+        char i = 'A';
+        char j = 'a';
+        String k = "Ala";
+        String l = "ma";
+        boolean m = true;
+        boolean n = false;
+        System.out.println("ab/b = " + ab / b);
+        System.out.println("i+j = " + i + j);
```

Comment:
A spróbuj dodać nawiasy w dodawaniu `i` oraz `j`  :)

Reply:
No to nie dodam charow tylko liczby z tablicy aski  ktore odpowiadaja A i a